wp-content/themes/core/components/routes/search/Search_Controller.php

```diff
@@ -68,15 +73,121 @@ public function render_breadcrumbs(): void {
 	}
 
 	/**
-	 * @return \Tribe\Project\Templates\Models\Breadcrumb[]
+	 * @return array
 	 */
-	protected function get_breadcrumbs(): array {
-		$page = get_option( 'page_for_posts' );
-		$url  = $page ? get_permalink( $page ) : home_url();
+	public function get_search_form_args(): array {
+		$args = [
+			Search_Form_Controller::CLASSES => [ 'c-search--full', 'search-results__form' ],
+			Search_Form_Controller::FORM_ID => uniqid( 's-' ),
+			Search_Form_Controller::VALUE   => get_search_query(),
+		];
+
+		return $args;
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_results_text_args(): array {
+		global $wp_query;
+		$total      = absint( $wp_query->found_posts );
+		$query_term = get_search_query();
+
+		if ( empty( $query_term ) ) {
+			return [];
+		}
+
+		$text = sprintf(
+			_n( 'Showing %d result for &lsquo;%s&rsquo;', 'Showing %d results for &lsquo;%s&rsquo;', $total, 'tribe' ),
+			$total,
+			$query_term
+		);
+
+		if ( 0 === $total ) {
+			$text = sprintf(
+				__( 'Your search for &lsquo;%s&rsquo; returned 0 results', 'tribe' ),
+				$query_term
+			);
+		}
 
 		return [
-			new Breadcrumb( $url, __( 'News', 'tribe' ) ),
+			Text_Controller::TAG     => 'p',
+			Text_Controller::CLASSES => [ 't-display-xx-small', 'search-results__summary' ],
+			Text_Controller::CONTENT => esc_html( $text ),
+		];
+	}
+
+	/**
```

Comment:
I'm confused what you are referring to here. Also, after I added your code suggestion of a return type, I'm getting a fatal error. Sorry, I don't really understand any of this type stuff in PHP. http://p.tri.be/gTWutu

Reply:
I took a stab at fixing this issue. Let me know what you think @defunctl!

Reply:
@tjdunklee looks like this broke the search acceptance tests. I'll take a look at that.

Reply:
Yea, this whole PR from the beginning broke that test. I know exactly what is happening, I just have no idea how to write a test. It is breaking because we removed the search form from the header. The search form is only on the search template now, so we should just be testing if the search works there.